### PR TITLE
[DEVELOPER 3307] Move inline scripts to libraries

### PR DESCRIPTION
**Pull request by ghost**
### JIRA Issue Link
* https://github.com/redhat-developer/developers.redhat.com/issues/3307

### Verification Process
1. Verify that the scripts moved still load correctly.

### CODE REVIEW

**Comment by rogeralloymagnetic** [comment]
retest this please

**Comment by jordanpagewhite** [comment]
retest this please

**Comment by robpblake** [comment]
@rogeralloymagnetic I don't see the DTM scripts loading after this change. Can you please take a look at verify.

cheers,

Rob

**Review by robpblake — CHANGES_REQUESTED** [review]
@rogeralloymagnetic As discussed, I don't see the DTM scripts loading after your changes.

**Review by jordanpagewhite — APPROVED** [review]
These changes look good to me. @KamiQuasi Could you give this PR a review when you have a chance since you have the most history with these scripts and their implementations? Thank you.

**Comment by robpblake** [comment]
retest this please

File: _docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhdp2.theme [hunk 1]
```diff
@@ -741,9 +741,9 @@ function rhdp2_preprocess_html(array &$variables) {
   $rhd_base_url = $rhd_config->get('rhd_base_url');
   $rhd_final_base_url = $rhd_config->get('rhd_final_base_url');
 
+  $variables['#attached']['drupalSettings']['rhdp2']['rhd_dtm_code'] = $dtm_code;
+  $variables['#attached']['drupalSettings']['rhdp2']['rhd_dtm_script'] = redhat_www_ddo_default(\Drupal::request()->attributes->get('node'));
   $variables['rhd_environment'] = $environment;
-  $variables['rhd_dtm_code'] = $dtm_code;
```
[the remark this answers is not in the document]
Comment (rogeralloymagnetic) [reply _docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhdp2.theme]: @jordanpagewhite the `rhd_dtm_code` is already added back in this commit: https://github.com/redhat-developer/developers.redhat.com/pull/3350/commits/51f2a794986586414b94bb60b70c371fc0efcf88.

The `rhd_dtm_script`  has been pulled out of twig and is now only used in the attached library.

[the remark this answers is not in the document]
Comment (jordanpagewhite) [reply _docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhdp2.theme]: Thank you Roger

**Comment by jordanpagewhite** [comment]
@robpblake Roger resolved the DTM script issue

Screen Shot 2019-12-16 at 12 18 05 PM

Could you re-review this when you have a chance? Sorry for having to ask. I know you're busy with other things now. I am unable to merge without you marking your request for change(s) resolved though. Thanks again! No rush.

**Review by robpblake — APPROVED** [review]
If @jordanpagewhite is happy, I'm happy.

**Pull request merged**